lib/ui/pages/views/home_view/home_view_model.dart

```diff
@@ -1,8 +1,55 @@
+import 'package:Covid19_PH/ui/partials/views/facilities_summary_view/facilities_summary_view.dart';
+import 'package:Covid19_PH/ui/partials/views/summary_view/summary_view.dart';
+import 'package:Covid19_PH/ui/partials/views/timeline_view/timeline_view.dart';
+import 'package:Covid19_PH/ui/partials/views/top_regions_view/top_regions_view.dart';
+import 'package:flutter/material.dart';
 import 'package:stacked/stacked.dart';
 
-class HomeViewModel extends BaseViewModel  {
-  initialise() {
-    print('Home View Model Initialized');
-    notifyListeners();
+class HomeViewModel extends BaseViewModel {
+  static final HomeViewModel _singleton = HomeViewModel._internal();
+  factory HomeViewModel() => _singleton;
+  HomeViewModel._internal();
+
+  // SummaryView summaryCard;
+  // TimelineView timelineCard;
+  // TopRegionsView topRegionsCard;
+  // FacilitiesSummaryView facilitiesSummaryCard;
+
+  initialise() {}
+
+  List<Widget> addCard(int id, Widget card) {
+    List<Widget> res = [];
+    // switch (id) {
```

Comment:
ViewModel must be completely logic and data, we should not handle widgets. It is recommend to avoid using functions specially in UI because it is not efficient and is not available for hot restart, instead convert it to class widget and put in a a seperate file.